qt/aqt/addons.py

```diff
@@ -104,6 +105,7 @@ class AddonMeta:
     max_point_version: int
     branch_index: int
     human_version: Optional[str]
+    update_enabled: bool
```

Comment:
Perhaps something like 'show_in_automatic_update_checks' might be clearer?

Reply:
Ah, I didn't realise it was also used there. In that case, it sounds like it's ok as is - thanks!